module.go

```diff
@@ -39,6 +39,7 @@ type (
 	}
 )
 
+// Open - open func
```

Comment:
If you can get throw the functions, could you add some more details to the comments? For example here, I guess open does a bit more than "open", or at least opens something specific!

Reply:
Good point. Should put more love in it....